{{cookiecutter.repo_name}}/package.json

```diff
@@ -444,7 +444,6 @@
       "at-rule-name-case": "lower",
       "at-rule-name-space-after": "always",
       "at-rule-no-vendor-prefix": true,
-      "stylelint-disable-reason": "always-before",
```

Comment:
> The following rules did not work well.
stylelint-disable-reason could not enforce providing a reason

Is all they seem to say. I was hit or miss on it personally.